.ruby-version

```diff
@@ -1 +1 @@
-2.2.3
+2.7.4
```

Comment:
I think we could actually remove Ruby from the dist entirely. It's not really used. It was just part of the standard setup stuff that Phil included.